src/components/App/App.css

```diff
@@ -1,14 +1,32 @@
+html {
+  background-attachment: fixed;
+  background-image: url('../../assets/fallColors-background.jpg');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: none;
+  padding: none;
+}
+
 #app {
+  color: #2c3e50;
   font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
   text-align: center;
-  color: #2c3e50;
   margin-top: 60px;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
 }
 
 .main {
-  align-items: center;
   display: flex;
   flex-direction: column;
+  align-items: center;
 }
+
+@media screen and (max-width: 600px) {
+  html {
+    background-attachment: fixed;
+    background-image: url('../../assets/vertical-background.jpg');
+    background-position: center;
```

Comment:
Still don't need the position and attachment properties since those aren't changing from the original styling of the html element.